packages/code-studio/src/main/AppRouter.tsx

```diff
@@ -7,16 +7,20 @@ import {
 } from 'react-router-dom';
 import AppInit from './AppInit';
 
-const basename = document.location.pathname;
+const StyleGuideRoot = React.lazy(() => import('../styleguide/StyleGuideRoot'));
+
+const baseURI = new URL(document.baseURI).pathname;
+
 function AppRouter(): ReactElement {
   return (
-    <Router basename={basename}>
+    <Router basename={baseURI}>
```

Comment:
Reviewing the [ReactRouter documentation](https://v5.reactrouter.com/web/api/BrowserRouter/basename-string): 
> A properly formatted basename should have a leading slash, but no trailing slash.

Right now, all of our `BASE_URL`'s defined do have a trailing slash (`./`, `/ide/`). I don't think this manifests any issues, and some of our code that uses the `baseURI` expects a trailing slash there... so I wouldn't bother changing anything right now.

Reply:
Oh, I missed that baseURI can have a trailing slash. I must have tested on a route that didn't

Reply:
Sure, that'd be fine. May as well match what the docs expect.

Digging in the code, it doesn't _really_ matter, the `basename` prop just gets passed through to `createBrowserHistory`, which actually removes the trailing slash: https://github.com/remix-run/history/blob/6104a6a2e40ae17a47a297621afff9a6cb184bfc/modules/createBrowserHistory.js#L51
Side note their code for removing the trailing slash is just:
```
export function stripTrailingSlash(path) {
  return path.charAt(path.length - 1) === '/' ? path.slice(0, -1) : path;
}
```